docs/manual/src/docs/asciidoc/_includes/servlet/oauth2/oauth2-resourceserver.adoc

```diff
@@ -3001,25 +3013,26 @@ fun rest(): RestTemplate {
 ====
 
 
-[NOTE]
-Unlike the {security-api-url}org/springframework/security/oauth2/client/OAuth2AuthorizedClientManager.html[OAuth 2.0 Authorized Client Manager], this filter interceptor makes no attempt to renew the token, should it be expired.
-To obtain this level of support, please create an interceptor using the <<oauth2client,OAuth 2.0 Authorized Client Manager>>.
+NOTE: Unlike the {security-api-url}org/springframework/security/oauth2/client/OAuth2AuthorizedClientManager.html[OAuth 2.0 Authorized Client Manager], this filter interceptor makes no attempt to renew the token, should it be expired.
+To obtain this level of support, create an interceptor by using the <<oauth2client,OAuth 2.0 Authorized Client Manager>>.
 
 [[oauth2resourceserver-bearertoken-failure]]
 === Bearer Token Failure
 
-A bearer token may be invalid for a number of reasons. For example, the token may no longer be active.
+A bearer token can be invalid for a number of reasons. For example, the token may no longer be active.
 
 In these circumstances, Resource Server throws an `InvalidBearerTokenException`.
 Like other exceptions, this results in an OAuth 2.0 Bearer Token error response:
 
-[source,http request]
+====
+[source,text]
```

Comment:
I think this should be formatted with `[source,http]`

Reply:
OK. Done.

Reply:
OK